app/utils/pages.tsx

```diff
@@ -10,26 +13,36 @@ interface IPage {
     path: string
 }
 
+export function getPathsToMap (){
+    const pathToLeagueData = path.join(process.cwd(), "app", "leagueConfiguration");
+    const pathsToMap = fs.readdirSync(pathToLeagueData);
+    return pathsToMap;
+}
+
+export function getConfigurationPage (filename:string){
```

Comment:
... I will explicitly forgive you for not having a type here 😛

Reply:
Hahahaha thank you. This will be typed later lololol